torch/distributed/fsdp/_fully_shard/_fully_shard.py

```diff
@@ -177,7 +177,8 @@ def fully_shard(
             which offers parameter/gradient/optimizer state offloading. See
             :class:`OffloadPolicy` and its subclasses for details.
         ignored_params: Optional(Set[nn.Parameter]): The set of parameters that we
-            don't want to shard with FSDP.
+            don't want to shard with FSDP. These parameters will not be moved to
+            the device by this function.
```

Comment:
suggestion: maybe change docstring to something like this?
"The set of parameters to be ignored by FSDP. They will not be sharded, moved to the device during init, nor have their gradients reduced in backward."